lib/stat_tracker.rb

```diff
@@ -1,11 +1,17 @@
 require 'csv'
-class StatTracker
 
-    def self.from_csv(locations)
-        holder = []
-        locations.each do |location|
-            holder << CSV.open(location[1], headers: true, header_converters: :symbol)
-        end
 
-    end
-end
\ No newline at end of file
+class StatTracker
+  def initialize(games, teams, game_teams)
+    @games = games
+    @teams = teams
+    @game_teams = game_teams
+  end
```

Comment:
Awesome, a helper method to divvy out the information to better manipulate I think is our next move!